src/server.cpp

```diff
@@ -2575,6 +2592,24 @@ void Server::DenySudoAccess(u16 peer_id)
 	Send(&pkt);
 }
 
+
+void Server::DenyAccessCompliant(u16 peer_id, u16 proto_ver, AccessDeniedCode reason,
```

Comment:
The main focus of the method lies on smartly chosing which packet to send for the given reason, depending of the protocol version of the client.